src/message.rs

```diff
@@ -1744,13 +1759,26 @@ pub async fn delete_msgs(context: &Context, msg_ids: &[MsgId]) -> Result<()> {
     }
     res?;
 
-    delete_msgs_locally_done(context, msg_ids, modified_chat_ids).await?;
+    if delete_for_all {
+        if let Some(any_chat_id) = modified_chat_ids.iter().next() {
```

Comment:
So, if someone tries to delete messages in multiple chats, then the deletion message will be sent only to one of them?

In this case, this should be documented in the UI and there should be an `ensure!(modified_chat_ids.len() <= 1);` or similar so that a proper error is returned.

Alternatively, make this a `while` instead of `if`

Reply:
i added a check that all messages are in the same chat and documented it like that. in practise, it will probably mostly like that, otherwise we can iterate